runtime/verified_gc/alloc.c

```diff
@@ -36,13 +31,14 @@ void *verified_allocate(unsigned long long wsize) {
     }
 
     verified_gc();
+
     mem = alloc(wsize);
     goto again;
   }
   return mem - 8U;
 }
 
-CAMLprim value caml_trigger_verified_gc(value v) {
+value verified_trigger_gc(value unit) {
```

Comment:
keep this as caml_trigger_verified_gc, else this won't be detected as a primitive (IINM) and we won't be able to use this directly from our code

Reply:
Makes sense. Thanks.